data/certChooser.js

```diff
@@ -62,3 +63,4 @@ function keyboardListener(e) {
 }
 
 document.addEventListener("keydown", keyboardListener);
+setTimeout(function() { document.getElementById("certPassword").focus(); }, 100);
```

Comment:
Rather than setTimeout, I would prefer calling focus as a result of some load or display event if possible. Would something like
`document.addEventListener("load", function() { document.getElementById("certPassword").focus(); });`
work?